integrationtests/test_base_set.py

```diff
@@ -22,13 +22,18 @@ def run_inference(config, checkpoint, image):
     return predictions
 
 
+@pytest.fixture(scope="session")
```

Comment:
is there a reason for using a function for a constant?

Reply:
I wanted to reuse the base path across the different test functions.

Reply:
@WanjiruCate just wondering why you are not using a constant, but as it works fine we can also leave it like this